src/Peachpie.Library.Graphics/PhpGd2.cs

```diff
@@ -667,16 +667,52 @@ public static long imagecolorallocatealpha(PhpResource im, int red, int green, i
         /// <summary>
         /// RGBA values to PHP Color format.
         /// </summary>
-        static long RGBA(long red, long green, long blue, long alpha = 0xff)
+        static long RGBA(long red, long green, long blue, long alpha = 0)
```

Comment:
is it reasonable to make the default value of `alpha` zero ? maybe rather don't make the argument options at all